src/exportsV2.js

```diff
@@ -0,0 +1,11 @@
+// the following will be injected in the app 'index.template.html', it contains
+// the graphql query results run in the server. The client will find them in
+// 'window.__APOLLO_STATE__' and uses them to initialize apollo-client cache,
+// thus removing the need to run the queries again in the client
+// https://ssr.vuejs.org/guide/build-config.html#manual-asset-injection
+module.exports.graphqlHtml = `
+    <% // added by 'quasar-app-extension-apollo' %>
+    <% if (ctx.mode.ssr) { %>
```

Comment:
I suggest to add ` && process.env.ssr` to the condition, to compile `renderState` only in case of server renderer, and not PWA `offline.html` page.